### PR TITLE
Update hworker dependencies

### PR DESCRIPTION
**Pull request by vshlapakov**
Review, please. // cc @dangra

---

This change is [Reviewable](https://reviewable.io/reviews/scrapinghub/kumo-stack-hworker/5)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,18 +3,16 @@ w3lib==1.13.0
...
 
-# Portia
-slybot==0.12.1
-scrapely==0.12.0
-
 # Scrapinghub
 hubstorage==0.22.0
-dateparser==0.3.2
+# dateparser==0.3.4 replaced with newer commit
+https://github.com/scrapinghub/dateparser/archive/1223377.zip/#egg=dateparser
```
Comment (vshlapakov) [inline requirements.txt]: Hworker dateparser version has a lot of changes compared with 0.3.4.

File: requirements.txt [hunk 2]
```diff
@@ -3,20 +3,18 @@ w3lib==1.13.0
 queuelib==1.4.2
 Scrapy==1.0.5
 
-# Portia
-slybot==0.12.1
-scrapely==0.12.0
```
Comment (dangra) [inline requirements.txt:8]: I think we better leave portia and scrapely, this is legacy stack after all and who knows what current spiders imports.

Comment (vshlapakov) [inline requirements.txt:8]: Sounds good

File: requirements.txt (hunk 1 above)
Comment (dangra) [inline requirements.txt]: dateparser 0.3.5 is out in pypi

Comment (vshlapakov) [inline requirements.txt]: Updated :+1:

Comment (dangra) [inline requirements.txt]: did you push?

Comment (vshlapakov) [inline requirements.txt]: Oh, have just pushed, sorry. I constantly forget that don't have write access to this repo, could you add me please?

**Pull request merged**